CONTRIBUTION.md

```diff
@@ -0,0 +1,82 @@
+### Recommended development flow
+
+The most natural way to develop che-machine-exec it's using Eclipse CHE workspace.
+To install Eclipse CHE you need to choose infrastructure(openshift, kubernetes or docker) and set up it.
+To create development Eclipse CHE workspace we provide che-machine-exec devfile [devfile.yaml](devfile.yaml).
+> See more about [devfile](https://redhat-developer.github.io/devfile)
+
+### Create Eclipse CHE workspace from devfile
+
+To start Eclipse CHE workspace, install the latest [chectl](https://github.com/che-incubator/chectl/releases) and start new workspace from devfile:
```

Comment:
I would like to mention how to do that from dashboard as well.

Reply:
preferable to link to https://www.eclipse.org/che/docs/che-7/che-quick-starts.html#running-che-locally_che-quick-starts (so that repo can change, etc)

Reply:
> preferable to link to https://www.eclipse.org/che/docs/che-7/che-quick-starts.html#running-che-locally_che-quick-starts (so that repo can change, etc)

Done.

> I would like to mention how to do that from dashboard as well.

I don't mind, but as for me current dashboard flow looks strange. To create new devfile using dashboard I  need to create new workspace from some random devfile and than replace content by che-machine-exec devfile. Looks like we missed up dashboard feature to simplify this flow.

Reply:
Not necessarily. One may use factory link